### PR TITLE
give projects access to project-<proj> provisionerId

### PR DESCRIPTION
**Pull request by djmitche**
This seems like a good idea based on https://docs.taskcluster.net/docs/manual/design/namespaces#projects

### CODE REVIEW

File: src/make-parameterized-roles.js [hunk 1]
```diff
@@ -42,6 +42,7 @@ module.exports.run = async function(options) {
       'project:<..>:*',
       'queue:get-artifact:project/<..>/*',
       'queue:route:index.project.<..>.*',
+      'queue:create-task:low:project-<..>/*',
```
Comment (tomprince) [inline src/make-parameterized-roles.js]: Any reason this isn't `highest`?

**Review by tomprince — COMMENTED** [review]
Does it make sense to also give out enough scopes to create workers for that use this provisioner ID?

File: src/make-parameterized-roles.js (hunk 1 above)
Comment (djmitche) [reply src/make-parameterized-roles.js]: Oh, that's a good point -- I copied it from somewhere without thinking much.

File: src/make-parameterized-roles.js [hunk 2]
```diff
@@ -42,6 +42,9 @@ module.exports.run = async function(options) {
       'project:<..>:*',
       'queue:get-artifact:project/<..>/*',
       'queue:route:index.project.<..>.*',
+      'queue:create-task:highest:project-<..>/*',
```
Comment (jonasfj) [inline src/make-parameterized-roles.js:45]: Give all priority levels, otherwise they can't give `queue:create-task:low:projec..` to a client/role

**Review by jonasfj — APPROVED** [review]

File: src/make-parameterized-roles.js (hunk 2 above)
Comment (djmitche) [reply src/make-parameterized-roles.js:45]: good point, thanks

**Pull request merged**